internal/accounts/create.go

```diff
@@ -134,3 +147,301 @@ func create(
 		include: createFlags.Include,
 	}, nil
 }
+
+func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow.Account, error) {
+	log := output.NewStdoutLogger(output.InfoLog)
+
+	log.Info(fmt.Sprintf("%s What name would you like to give this new account?", output.GoEmoji()))
+	name := output.NamePrompt()
+
+	networkName, selectedNetwork := output.CreateAccountNetworkPrompt()
+	// create new gateway based on chosen network
+	gw, err := gateway.NewGrpcGateway(selectedNetwork.Host)
+	if err != nil {
+		return nil, err
+	}
+	privateJsonFileName := ""
+	saveToGitIgnore := false
+	if selectedNetwork != config.DefaultEmulatorNetwork() {
+		privateJsonFileName = fmt.Sprintf("%s.private.json", name)
+		log.Info(fmt.Sprintf("%s For security purposes, the private key generated for this account will be "+
+			"stored in separate file: %s", output.WarningEmoji(), privateJsonFileName))
+		saveToGitIgnore = output.AddToGitIgnorePrompt(privateJsonFileName)
+	}
+	log.Info(fmt.Sprintf("%s This command will perform the following:\n", output.WarningEmoji()))
+	log.Info(fmt.Sprintf("- Generate a new ECDSA P-256 public and private key pair\n"))
+	if selectedNetwork != config.DefaultEmulatorNetwork() {
+		if saveToGitIgnore {
+			log.Info(fmt.Sprintf("- Save the private key to %s\n", privateJsonFileName))
+		}
+	}
+	log.Info(fmt.Sprintf("- Create a new account on %s and pair the public key with the new account\n", networkName))
+	log.Info(fmt.Sprintf("- Save the newly created account configuration to flow.json\n"))
+	output.NextStepPrompt()
+
+	service := services.NewServices(gw, state, output.NewStdoutLogger(output.NoneLog))
+
+	log.Info(fmt.Sprintf("%s Generating public and private keys for %s", output.WarningEmoji(), name))
+
+	key, err := service.Keys.Generate("", crypto.ECDSA_P256)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info(fmt.Sprintf("%s Successfully generated public and private keys", output.OkEmoji()))
```

Comment:
I don't think we need to have two outputs for the generation of keys:
(Also the extra continue is confusing and not needed)
```
⚠️ Generating public and private keys for Foo
✅ Successfully generated public and private keys
Use the arrow keys to navigate: ↓ ↑ → ←
? Press <ENTER> to continue:
  ▸ CONTINUE
```

This should just be:
```
Successfully generated public and private keys
```

Reply:
addressed